Source/DafnyLanguageServer.Test/Unit/NewtonsoftConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Unit {
+
+  [TestClass]
+  public class NewtonsoftConfiguration {
+    private string GetLibraryVersion(string csprojFile, string library) {
+      var libraryVersionExtractor = new Regex(
+        $"Include=\"{Regex.Escape(library)}\" Version=\"([^\"]+)\"");
+      var csprojContent = System.IO.File.ReadAllText(csprojFile);
+      var libraryVersion = libraryVersionExtractor.Match(csprojContent);
+      Assert.IsTrue(libraryVersion.Success, $"Could not extract {library}'s version from {csprojFile}");
+      return libraryVersion.Groups[1].ToString();
+    }
+
+    [TestMethod]
+    public void NewtonsoftVersionMatch() {
+      var dlsNewtonsoftVersion = GetLibraryVersion(@"../../../../DafnyLanguageServer/DafnyLanguageServer.csproj", "Newtonsoft.Json");
```

Comment:
I don't understand how this relative path works. I thought you can't assume anything about the working directory when running tests. Can you?

Reply:
It looks like it work ! It assumes we are in `DafnyLanguageServer.Test\bin\Debug\net6.0`

Reply:
It's definitely better not to assume anything about the output directory, in which tests run (and I worked really hard not to in the xUnit lit test runner), but in a test that will fail loudly if the assumption is ever violated, I think it's fine.